src/main/drivers/timer.h

```diff
@@ -63,15 +63,16 @@ typedef uint32_t timCNT_t;
 #endif
 
 typedef enum {
-    TIM_USE_ANY           = 0x0,
-    TIM_USE_NONE          = 0x0,
-    TIM_USE_PPM           = 0x1,
-    TIM_USE_PWM           = 0x2,
-    TIM_USE_MOTOR         = 0x4,
-    TIM_USE_SERVO         = 0x8,
-    TIM_USE_LED           = 0x10,
-    TIM_USE_TRANSPONDER   = 0x20,
-    TIM_USE_BEEPER        = 0x40
+    TIM_USE_ANY            = 0x0,
+    TIM_USE_NONE           = 0x0,
+    TIM_USE_PPM            = 0x1,
+    TIM_USE_PWM            = 0x2,
+    TIM_USE_MOTOR          = 0x4,
+    TIM_USE_SERVO          = 0x8,
+    TIM_USE_LED            = 0x10,
+    TIM_USE_TRANSPONDER    = 0x20,
+    TIM_USE_BEEPER         = 0x40,
+    TIM_USE_CAMERA_CONTROL = 0x40,
```

Comment:
Is it intended that beeper and camera control have the same value?

Reply:
No. Fixing.